1/src/_backend/seeds.js

```diff
@@ -0,0 +1,98 @@
+  const { setDriver }= require('../../src/_backend/server');
+
+let data = require('./database');
+
```

Comment:
I think you should remove the empty line and reformat code for this file. Then we will have a great look